### ISSUE TITLE
[Bug]: No status icon in GNOME 3 (Debian 11)

### ISSUE THREAD

**Issue by marbx** [nextcloud/desktop#4752]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The NextClold system tray icon doesn't appear at all on Debian 11

### Steps to reproduce

- Installed Debian 11, all settings left as default, resuting in GNOME 3, Wayland
- Installed Nextcloud via "Software"
- Logged into NextCloud
- Enabled auto-start
- Rebooted several times, waited several days.
- NextCloud seems to run in the background, but there is no system tray icon.

### Expected behavior

A NextCloud icon (e.g. the green tick mark) appears in the system tray

### Which files are affected by this bug

The Debian system tray

### Operating system

Linux

### Which version of the operating system you are running.

Debian 11

### Package

Distro package manager

### Nextcloud Server version

23.0.6

### Nextcloud Desktop Client version

3.1.1 (Debian)

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Additional info

Clicking on "Create Debug Archive" has no effect

**Comment by claucambra** [comment]
Hi, what desktop environment are you using? If it is GNOME (or another DE that does not display app indicators by default), do you have an extension that enabled app indicators to be displayed? Thanks!

**Comment by marbx** [comment]
Hi @claucambra 
thank you! And yes, GNOME is the Debian 11 default desktop environment. 

Regarding extensions: my issue is that NextCloud requires the end user to perform additional installations. 

I now have read the GNOME blog of (2017-08-31 about status icons](https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome) and found the [status of the NextCloud Status Icon Migration](https://wiki.gnome.org/Initiatives/StatusIconMigration) to be "Standard behavior is fine without the status icon, since the window can be accessed via the launcher. Integration via libcloudprovider is also planned for GNOME 3.26, and this should provide access to the window."

GNOME 3.26 has been released in September 2017.


As I now realize this is a GNOME issue, and not a Debian issue, I changed the title and close this issue as a duplicate of issue #2217 which mentioned GNOME 3.36.3 in July 2020
